app/scripts/main.js

```diff
@@ -665,6 +733,7 @@ document.addEventListener("DOMContentLoaded", function () {
 
         showLoading();
 
+        //the credentials are added during the createDatabase step
         createDatabase(formValues)
             .then(getUser)
             .then(uploadConfigFile)
```

Comment:
You converted it to add-in. You can use standard add-in intialization methods like we do in export add-in:
```
geotab.addin.registrationConfig = function () {
    let addin: Addin;

    return {
        initialize: (api, state, callback) => {
            addin = new Addin(api);
            callback();
        },
        focus: () => {
            addin.render();
        },
        blur: () => {
            addin.unload();
        }
    };
};
```
In such case you'll already have API object. So you can get rid of geotab.api.js file

Reply:
yes clear but this is a later update to be executed...possibly